exchanges/huobihadax/huobihadax_wrapper.go

```diff
@@ -191,7 +191,7 @@ func (h *HUOBIHADAX) FetchTradablePairs(asset asset.Item) ([]string, error) {
 
 	var pairs []string
 	for x := range symbols {
-		pairs = append(pairs, symbols[x].BaseCurrency+"-"+symbols[x].QuoteCurrency)
+		pairs = append(pairs, fmt.Sprintf("%v%v%v", symbols[x].BaseCurrency, h.CurrencyPairs.ConfigFormat.Delimiter, symbols[x].QuoteCurrency))
```

Comment:
Should use h.GetPairFormat(asset, false) here. Will stop reporting these from now one but basically whenever you're wanting to get the request/config format.